app/WebParsing/ParsecCombinators.hs

```diff
@@ -105,40 +105,18 @@ parseUntil parser = P.manyTill P.anyChar (P.try parser)
 splitPrereqText :: Parser [String]
 splitPrereqText = do
     P.manyTill ((P.try parseNotes) <|> (P.try parseNoteLine) <|>
-        (P.try (parseCategory False)) <|> (parseUntil P.eof)) P.eof
+        (P.try parseCategory) <|> (parseUntil P.eof)) P.eof
```

Comment:
You shouldn't need parentheses around `P.try parseCategory` or `parseUntil P.eof`